### DISCUSSION TITLE
UrsaWindow标题栏的按钮没办法被点击，感觉被击穿了

### DISCUSSION THREAD

**Discussion by xiaoCuiNice** [irihitech/Ursa.Avalonia#477] [Q&A]
标题栏的控件无法被选中，双击按钮变成了双击标题栏，最大化了
![21213](https://github.com/user-attachments/assets/37729ae0-2734-4c31-aad1-7f803dafea60)

**Comment by xiaoCuiNice** [comment]
按钮不在MainWindow，在MainView中，我知道在MainWindow可以解决，有LeftContent属性，但是考虑在android、浏览器运行，是没有MainWindow的，有没有什么解决方案呢大佬们

**Comment by rabbitism** [comment]
如果不需要TitleBar的这些功能可以直接用普通的Window，不需要用UrsaWindow的
